Documentation/Events/EventDispatcher/Index.rst

```diff
@@ -281,3 +281,21 @@ are marked as :php:`@internal`, just like other non-API parts of TYPO3, but :php
 avoided whenever technically possible.
 
 
+.. index:: Event listener; Best practices
+.. _EventDebugging:
+
+Debugging event handling
+========================
+
+A complete list of all registered event listeners can be viewed in the the module 
+:guilabel:`System > Configuration > Event Listeners (PSR-14)`. The system extension
+`lowlevel` has to be installed for this module to be available.
+
+.. TODO: add screenshot
```

Comment:
Should this be a draft until these where added?

Reply:
I do not have time to add them for now. But I think the information is valid and helpful even before the screenshots are added. So I would like to add the PR like this

Reply:
Alright, fine with me. :+1: